Sources/PubNubChatComponents/Message List/MessageListViewController.swift

```diff
@@ -133,4 +209,162 @@ open class ChatViewController: CollectionViewComponent {
   fileprivate var isUserInitiatedScrolling: Bool {
     return collectionView.isDragging || collectionView.isDecelerating
   }
+  
+  // MARK: UIViewControllerTransitioningDelegate
+
+  @objc(presentationControllerForPresentedViewController:presentingViewController:sourceViewController:)
+  public func presentationController(
+    forPresented presented: UIViewController,
+    presenting: UIViewController?,
+    source: UIViewController
+  ) -> UIPresentationController? {
+    
+    return AddMessageReaction.PresentationController(
```

Comment:
return is not needed

Reply:
I added a `guard` statement to check whether the `gestureRecognizedCell` is not nil (you also suggested to check it) so I had to leave a `return` statement. Let me know if it's fine for you.